src/Bonnier/WP/Cxense/Assets/Scripts.php

```diff
@@ -108,7 +108,7 @@ private function objects_to_array($items)
 
             // Only one? Just return it
             if(count($items) <= 1) {
```

Comment:
There's a check before `objects_to_array` is called if it's empty or not. But good point. I need also to add additional check if `$items` is empty